irods/test/scripts/pp_inline.py

```diff
@@ -0,0 +1,32 @@
+from irods.session import iRODSSession
+import irods.client_configuration as cfg
+
+## -- Only for forcing 4.3 servers into obeying the old auth protocol & APIs
+# cfg.legacy_auth.force_legacy_auth = True
```

Comment:
Is this needed?

Reply:
This script will also be replaced by a real test, and - no -force_legacy_auth (which I mean to document in the README) will be only for folks who for whatever reason elect to use that `pam` legacy code path in the server.